tools/system_libs.py

```diff
@@ -961,7 +962,6 @@ class libcxx(NoExceptLibrary, MTLibrary):
 class libunwind(NoExceptLibrary, MTLibrary):
   name = 'libunwind'
   cflags = ['-Oz', '-D_LIBUNWIND_DISABLE_VISIBILITY_ANNOTATIONS']
-  src_dir = ['system', 'lib', 'libunwind', 'src']
```

Comment:
Do we not need this anymore? Can we specify where Unwind-wasm.c is without this?

Reply:
Good catch yes, that was not indended.